src/components/generators/view-components/string.vue

```diff
@@ -54,4 +97,38 @@ export default {
 .fa-external-link:hover {
   cursor: pointer;
 }
+
+.toast-container {
+  display: block;
+  max-width: 350px;
```

Comment:
Using rems is better for responsiveness.

Reply:
used 25rem